docs/modules/ROOT/pages/integrating-with-existing-deployments-collapsed.adoc

```diff
@@ -0,0 +1,252 @@
+. **Prometheus** - this should already be configured as a Data Source.
+. **Alertmanager** - this should already be configured as a Data Source. 
+  However, some of the dashboards query the Alertmanager API and so you will need to add a new Data Source in the same way as you did for the Cluster Monitor.
+  - This should be named _Alertmanager API_, also of type _JSON API_, and the URL should point to your Alertmanager instance. 
+  - Configure any authentication as needed.
+
+If you are utilizing Grafana provisioning for dashboards, and would instead like to specify this in a configuration file, then your `grafana.yml` should look something like this:
+[source, console]  
+----
+...
+   - name: JSON API
+      type: marcusolsson-json-datasource
+      url: http://<Cluster Monitor URL>:7196/api/v1
+      basicAuth: true
+      basicAuthUser: <Cluster Monitor Username>
+      basicAuthPassword: <Cluster Monitor Password>
+    - name: Alertmanager API
+      type: marcusolsson-json-datasource
+      url: http://<Alertmanager URL>/api/v2
+      // Configure basicAuth as needed.
+...
+---- 
+
+=== Prometheus: Scrape config
+
+You will need to modify your existing Prometheus configuration file (typically named `prometheus.yml`).
+Add in a `scrape_config` job for your Couchbase Server nodes:
+
+.prometheus.yml
+[source, yaml]
+----
+global:
+  scrape_interval: 30s
+
+scrape_configs:
+  - job_name: couchbase-server
+    basic_auth:
+      username: Administrator <.>
+      password: password <.>
+    static_configs:
+      - targets:
+          - 10.145.212.101:8091 <.>
+          - 10.145.212.102:8091
+          - 10.145.212.103:8091
+          ...
+        labels:
+          cluster: "Couchbase Server Cluster" <.>
+----
+
+<.> `username` should be set to a Couchbase Server user with at least "External Stats Reader" permission.
+<.> `password` should be the password of the above user.
+<.> `targets` should specify the addresses of all the nodes in your cluster.
+    The port number needs to be the management port - use `:18091` if you have TLS enabled.
+    There are alternative configuration options that do not require hard-coding the addresses - refer to the https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config[Prometheus documentation^] for more details.
+<.> Any labels defined here will be added to **all** metrics from this cluster - you may want to configure e.g., `environment`, `datacenter` etc.
+    The label `cluster` is required, and should match the name the cluster is configured to use in the Web Console.
+
+Add another `scrape_config` job for the previously-installed Cluster Monitor:
+
+.prometheus.yml
+[source, yaml]
+----
+...
+- job_name: couchbase-cluster-monitor
+      basic_auth:
+        username: admin  <.>
+        password: password <.>
+      metrics_path: /api/v1/_prometheus <.>
+      static_configs:
+          - targets: [10.145.212.110:7196] <.>
```

Comment:
Should this be a CB IP? Should we have something like `<Cluster Monitor URL>` instead?

Reply:
It's just an example of an internal IP, but yes that is possible